pycrust crashes

### Test script
```python
import wx
import wx.py

app = wx.App()
win = wx.Frame(None)
win.Show()
py_crust = wx.py.crust.CrustFrame(win)
py_crust.Show()
app.MainLoop()
```
Process finished with exit code -1073741819 (0xC0000005)

### Environment
- Windows 10 64Bit
- Python 3.6.1 (v3.6.1:69c0db5, Mar 21 2017, 17:54:52) [MSC v.1900 32 bit (Intel)] on win32
- wxPython 4.0.0a3.dev3091+8bed348 msw (phoenix)

### Findings
There seems to be a problem with the root item.
[Line 301 in filling.py ](https://github.com/wxWidgets/Phoenix/blob/master/wx/py/filling.py#L301)(`self.tree.SelectItem(self.tree.root)`) results in crash and when commented out, [line 226 in filling.py](https://github.com/wxWidgets/Phoenix/blob/master/wx/py/filling.py#L226) (`parent == self.root`) crashes.  (`parent != self.root` works, but not `parent == self.root`)

I have updated to 4.0.0a4.dev3141+18e35f5 msw (phoenix). Now i get the following exception when i run the test script:
```python
Traceback (most recent call last):
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 95, in OnSelChanged
    self.display()
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 172, in display
    text += self.getFullName(item)
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 208, in getFullName
    obj = self.GetItemData(parent)
wx._core.wxAssertionError: C++ assertion "item.IsOk()" failed at ..\..\src\msw\treectrl.cpp(1032) in wxTreeCtrl::GetItemParam(): invalid tree item
Traceback (most recent call last):
  File "D:/devel/Small Test Apps/py-crust.py", line 7, in <module>
    py_crust = wx.py.crust.CrustFrame(win)
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\crust.py", line 336, in __init__
    *args, **kwds)
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\crust.py", line 57, in __init__
    rootIsNamespace=rootIsNamespace)
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 302, in __init__
    self.tree.display()
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 172, in display
    text += self.getFullName(item)
  File "D:\devel\ATLogger3\python36-32\lib\site-packages\wx\py\filling.py", line 208, in getFullName
    obj = self.GetItemData(parent)
wx._core.wxAssertionError: C++ assertion "item.IsOk()" failed at ..\..\src\msw\treectrl.cpp(1032) in wxTreeCtrl::GetItemParam(): invalid tree item
```

--- a/etg/treectrl.py
+++ b/etg/treectrl.py
@@ -45,7 +45,7 @@ def run():
     c.addCppMethod('bool', '__eq__', '(const wxTreeItemId& other)', """\
         return *self == *other;
         """)
-    c.addCppMethod('bool', '__neq__', '(const wxTreeItemId& other)', """\
+    c.addCppMethod('bool', '__ne__', '(const wxTreeItemId& other)', """\
         return *self != *other;
         """)
 

